Arquivos de contribuição atualizados

antoniamaia opened the pull request:
closes #86

antoniamaia committed:
adicionando contributing ao readme

antoniamaia committed:
Atualizar arquivos de Contribuição

antoniamaia commented:
@camilamaia 👀

camilamaia commented on README.md:
```diff
@@ -127,3 +127,12 @@ Gera um CNPJ válido aleatório.
+
+## Contributing
+
+Sua colaboração é sempre bem-vinda! Preparamos o [ arquivo contributing][contributing] pra te ajudar nos primeiros passos. Toda ajuda conta! Sinta-se livre para criar novas [GitHub issues][github-issues] e interagir aqui.
```
```suggestion
Sua colaboração é sempre bem-vinda! Preparamos o [arquivo contributing][contributing] pra te ajudar nos primeiros passos. Toda ajuda conta! Sinta-se livre para criar novas [GitHub issues][github-issues] e interagir aqui.
```

camilamaia reviewed (approved):
Mandou bem, ficou muito ⭐ 

Uma pequena sugestão para ajudar a deixar o PR mais claro para quem vai revisar. O título poderia ser algo como:

```Criação de arquivo de contribuição em Português.```

E a descrição:

```
* Movi o arquivo de contribuição em inglês para CONTRIBUTING_EN.md
* Criei um novo arquivo chamado CONTRIBUTING.md, contendo as instruções em Português.
* Adicionei uma seção aos arquivos README (README.md README_EN.md) contendo instruções sobre como contribuir

Closes #86
```

Mas são só detalhes, aos poucos vai ficando mais natural fazer esse tipo de coisa :)

Fiz uma mini sugestão no "código". Só adicioná-la que tá pronto para mergear.

antoniamaia committed:
Update README.md

Pequena alteração sugerida no texto, tirando o espaço a mais que passou despercebido.

Co-authored-by: Camila Maia <cmaiacd@gmail.com>

camilamaia reviewed (approved):
Uhull!
